packages/datetime/src/_daterangepicker.scss

```diff
@@ -63,20 +63,45 @@ Styleguide components.datetime.daterangepicker.js
     margin-right: $pt-grid-size;
   }
 
+  .DayPicker-Day--hovered-range:not(.DayPicker-Day--outside) {
+    border-radius: 0;
+
+    // need to disable hover-styles for all variants of selected dates
+    // stylelint-disable max-line-length
+    &:not(.DayPicker-Day--selected):not(.DayPicker-Day--selected-range):not(.DayPicker-Day--selected-range-start):not(.DayPicker-Day--selected-range-end) {
+      background-color: $light-gray4;
+
+      &:hover {
+        background-color: $light-gray1;
+      }
+    }
+    // stylelint-enable max-line-length
+  }
+
   .DayPicker-Day--selected-range:not(.DayPicker-Day--outside) {
     border-radius: 0;
     background-color: $light-gray4;
+
+    &:hover {
+      background-color: $light-gray1;
+    }
+  }
+
+  .DayPicker-Day--outside {
```

Comment:
I'd be down with removing them. They are only useful if you can see one month at a time.